client/components/domains/transfer-domain-step/transfer-restriction-message.jsx

```diff
@@ -0,0 +1,133 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize, moment } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormattedHeader from 'components/formatted-header';
+import support from 'lib/url/support';
+import paths from 'my-sites/domains/paths';
+
+class TransferRestrictionMessage extends React.PureComponent {
+	static propTypes = {
+		creationDate: PropTypes.string,
+		domain: PropTypes.string,
+		loading: PropTypes.bool,
+		termMaximumInYears: PropTypes.number,
+		transferEligibleDate: PropTypes.string,
+		transferRestrictionStatus: PropTypes.string,
+	};
+
+	goToMapDomainStep = event => {
```

Comment:
There is no `selectedSite` prop here.

Reply:
Ack. Missed that when moving to the new component.